packages/http-smoke-testing/src/RouterAdapter/SymfonyRouterAdapter.php

```diff
@@ -27,8 +33,25 @@ public function __construct(RouterInterface $router)
     public function getAllRouteInfo()
     {
         $allRouteInfo = [];
+        $annotationReader = new AnnotationReader();
         foreach ($this->router->getRouteCollection() as $routeName => $route) {
-            $allRouteInfo[] = new RouteInfo($routeName, $route);
+            $annotations = [];
+
+            if ($route->hasDefault('_controller')) {
+                try {
+                    $reflectionMethod = new \ReflectionMethod($route->getDefault('_controller'));
+
+                    foreach ($annotationReader->getMethodAnnotations($reflectionMethod) as $annotation) {
+                        if ($annotation instanceof DataSet || $annotation instanceof Skipped) {
+                            $annotations[] = $annotation;
+                        }
+                    }
+                } catch (\ReflectionException $exception) {
+                    // Just could not parse the reflection. Do noting.
+                }
+            }
```

Comment:
I would like to se this block moved in its own method (or maybe 2 methods).

If it is against your will we can manage it by ourselves.

Reply:
Or maybe according to https://github.com/shopsys/shopsys/pull/1451#discussion_r341939667 only getting all annotations might be extracted.

Reply:
Moved the annotation extraction into two separate private methods.